components/TextFields/src/ContainedInputView/UIControlState+TextControl.h

```diff
@@ -0,0 +1,24 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+
+typedef UIControlState MDCTextControlState;
```

Comment:
As noted in the PR description, I'm calling it MDCTextControlState instead of MDCBaseTextFieldState because things like MDCFilledTextFields, MDCOutlinedTextFields, and all the input chip view and text area classes will use it too.

Reply:
lgtm